microphone/drv_audio_pdm.h

```diff
@@ -8,7 +8,7 @@
 
 #define PDM_BUF_NUM 	2
 #define PDM_BUF_SIZE 	2048
-#define DECIMATION		32
+#define DECIMATION		8 
```

Comment:
See main comment, about how high the decimation should be

Reply:
`LOW` targeted 625 Hz before, why is the requirement changing to target  ~1250 Hz? Where does 1250 Hz come from?

Reply:
to be clear, only frequencies allowed would be the result of dividing the PDM clock by and integer, a nice split of 2048 being better,  doing some more precise calculations, with our config:

x = PDM_CLOCK = (32.768/ 30)/64 = 17.066 KHz

Some possible decimation factors
x/16 ~ 1.066 KHz 
x/14 ~ 1.219 KHz
x/13 ~ 1.312 KHz
...
x/8 ~ 2.133 KHz

Reply:
I'm still unsure on what do here, the 1250Hz with 16 bit sample depth decoding script worked fine with some the raw conflab audio files. That means the midge was flashed to do that. I guess with the undocumented clock frequency flag. Is that something that we can replicate or as you say it needs a modified SDK?

Reply:
I can try replicating, but do we actually want to complicate the SDK setup process just to keep a sampling frequency for which we have no other justification than "that's what it was". If 1KHz could work fine, why complicate the setup process just to keep 1.250KHz?

Reply:
Perhaps this is a topic to discuss in the biweekly (? What are the requirements for the LOW frequency and such

Reply:
Fair point about complicating the SDK setup. In this case, I think we should go for:
low frequency: 1000 Hz and high frequency 16000 Hz, which is `PDM_CLK_DIV = 32` and `DECIMATION = 16`. As this gives us nice rounded numbers. I'll check in with Hayley to make sure she's happy with this change.

I also realised that Audacity supports inputting any value for the sample rate, if you just edit the text field.

Reply:
Need to dig a little bit more as it seems some of the stuff I remember from the SDK could be wrong? 
Digging the current repo I can see the config function directly writing the value for 20KHz in the register, but I remember there being a function that did a precheck in the middle (? Also I remember that bosco saw the register falling back to a default value when using the code from Ioannis? Anyways, will dig some more to get a better understanding

Reply:
Amazing, thanks a lot for getting to the bottom on this.

Reply:
We talked about this in the meeting. For the current version of the SDK, we do not need to edit it. Jose already checked that the register is not set back to the default value and the audio files can be decoded with the frequencies that we are setting. Please mark as resolved after you read this.